defaults/main.yml

```diff
@@ -49,3 +50,11 @@ tomcat_users:
   - name: admin
     password: must-be-changed
     roles: manager-gui
+
+## Deploy
+tomcat_download_wars: true
+tomcat_war_to_deploy_urls:
+  - https://tomcat.apache.org/tomcat-8.5-doc/appdev/sample/sample.war
```

Comment:
shouldn't this variable to be in the test folder instead of in the default folder?